### ISSUE TITLE
8/8 task

### ISSUE THREAD

**Issue by hsh6449** [hsh6449/detectron2_seg#3]
# 하이퍼 파라미터 조정중

- 기존 연구노트에 50만번 시행 후 결과가 안나와 있어서 50만번 돌리는 중 -> 내일와서 과적합 확인
- train set과 validation set split
- ITER 57000번 현재 loss는 0.055~ 0.067 사이(5만번만 해도 충분해 보이긴함)

8/9 해볼것 : Batch size조정에 따른 성능 차이([train,val] set이용 + iter는 5만번 ), 다른 모델 적용 가능한지 검토

**Comment by hsh6449** [comment]
container를 실수로 끄는 바람에 다시 시작,.,,
- LR : 0.0005, batchsize : 10, ITER : 50만번
- train set으로 학습중 -> val set으로 과적합 확인 가능

**Comment by hsh6449** [comment]
1. 8/12 To do
- PointRend로 학습시키니까 훨씬 더 빠르게 loss가 감소 
- LR을 0.0005 -> 0.0001로 낮춰서 해보자
- evaluation 하는 코드 작성 -> 모델 비교(#4 class 문제 해결하기..)

2. Plan
- Mask 시각화 코드 작성
